### PR TITLE
Dont format files in TestData

### PR DESCRIPTION
**Pull request by tkatzir**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/39095

## Description
Don't format files in the TestData folder (as well as other folders, already in the blacklist.)

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: demisto_sdk/commands/format/format_module.py [hunk 1]
```diff
@@ -114,7 +114,7 @@ def format_manager(input: str = None,
                 current_excluded_files.remove('CommonServerPython.py')
             if os.path.basename(file_path) in current_excluded_files:
                 continue
-            if dirname.endswith('test_data') or dirname.endswith('doc_imgs'):
+            if dirname.lower().endswith('test_data') or dirname.lower().endswith('testdata') or dirname.lower().endswith('doc_imgs'):
```
Comment (roysagi) [inline demisto_sdk/commands/format/format_module.py:117]: we actually have that in validate:
`any(test_dir in str(file_path) for test_dir in TESTS_AND_DOC_DIRECTORIES):`
better use this here as well.

**Review by roysagi — CHANGES_REQUESTED** [review]

File: demisto_sdk/commands/format/format_module.py (hunk 1 above)
Comment (tkatzir) [reply demisto_sdk/commands/format/format_module.py]: Fixed by https://github.com/demisto/demisto-sdk/pull/1465/commits/d832924233566e6e4b937a7d8f30e9956eca4374

**Review by roysagi — APPROVED** [review]
If you could please add a changelog entry

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,7 @@
   - Fixed an issue where the **update-release-notes** command did not add docker image release notes entry for release notes file if a script was changed.
   - Fixed an issue where the **update-release-notes** command did not detect changed files that had the same name.
   - Fixed an issue in the **update-release-notes** command where the version support of JSON files was mishandled.
+* Fixed **format** to skip files in `TESTS_AND_DOC_DIRECTORIES`
```
Comment (roysagi) [inline CHANGELOG.md:18]:
```suggestion
* Fixed an issue where **format** did not skip files in test and documentation directories.
```

**Review by yuvalbenshalom — APPROVED** [review]

**Pull request merged**